plugin/vimux.vim

```diff
@@ -96,7 +96,7 @@ endfunction
 
 function! VimuxOpenRunner() abort
   let existingId = s:existingRunnerId()
-  if existingId !=# ''
+  if !exists('g:VimuxRunnerIndex') && existingId !=# ''
```

Comment:
How about calling `s:hasRunner()`?

Reply:
Ah sorry I'm thinking of the version I've got sitting in #237. I'll take a closer look at this.